client/src/components/Settings/EmailForm.js

```diff
@@ -26,8 +26,8 @@ class EmailForm extends React.Component {
         const { classes } = this.props;
 
         return(
-                <div>
-                    <Typography variant='h6'>Current email address:</Typography>
+                <div className={classes.emailContainer}>
```

Comment:
Maybe add https://material-ui.com/demos/text-fields/ This might help give more cleaner asthetic and draw the bounderies around those areas

Reply:
I added a border around the entire Settings page, but I don't think that displaying the settings in a text field is currently necessary, at least not until those fields can be updated. I think having a disabled text field is useful when a field is disabled due to the selection of other options; however, when the purpose is solely to display information, I think that the email address and phone number should still be shown as pure text.